orchestration/src/main/resources/spec/orchestration.yaml

```diff
@@ -555,6 +658,163 @@ components:
         - profile-sexual-orientation
         - profile-trade-union
         - profile-sensitive-data
+
+    # --- Grounding Module ---
+    GroundingModuleConfig:
+      type: object
+      required:
+        - type
+        - config
+      properties:
+        type:
+          type: string
+          anyOf:
+            - enum:
+                - document_grounding_service
+            - {}
```

Comment:
(Comment/Question)
wtf is this.

Reply:
This is done for future extensibility. Without the anyof, users may complain if the enum allows additional values in the future. Not sure how strong this argument really is though..

Reply:
See also this (very related) carfuffle: https://github.com/OpenAPITools/openapi-diff/issues/303

Reply:
yeah, we are also setting the flag `enumUnknownDefaultCase` for exactly this reason..